assemblies/web-console-guide/assembly_asset-generation-ui.adoc

```diff
@@ -0,0 +1,53 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2025-05-28
+
+ifdef::context[:parent-context-of-asset-generation-ui: {context}]
+
+:_mod-docs-content-type: ASSEMBLY
+
+ifndef::context[]
+[id="asset-generation-ui"]
+endif::[]
+ifdef::context[]
+[id="asset-generation-ui_{context}"]
+endif::[]
+= Generating assets for application deployment in {WebName}
+
+:context: asset-generation-ui
+
+[role=_abstract]
+Starting with 8.0.0, asset generation in {ProductShortName} {WebName} allows you to create different  assets (manifests) required to deploy applications from Cloud Foundry (CF) instances to {ocp-full} or Kubernetes. 
+
+To generate assets, you must become familiar with the following entities:
+
+* Archetypes - Archetypes contain one or more criteria tags to identify technologies such as EAP 6. An archetype helps you to classify applications based on technologies common to them. For asset generation, you can plan your migration paths by archetypes because you configure the other asset generation entities such as target profiles that contain the generators in the archetype. 
+
+* Generators - A generator takes a templating engine, such as Helm, along with a set of templates as input configuration. Generators use the templates that are defined for a target platform and a migration path to generate deployment assets. When you run asset generation process, the generator parses the application discovery manifest to create deployment assets.
++
+For example, the migration path for an EAP 6 application is EAP 8 and the target platform is Kubernetes. A generator can generate the required assets to migrate the EAP 6 application to EAP 8 that is to be deployed on Kubernetes. The assets are generated by parsing the discovery manifest of the EAP 6 application.
+
+* Target profiles - A target profile represents the target platform to which you want to migrate. It contains a seqeuence of generators. Through target profiles, you can run more than one generator, with each generator creating assets for a specific migration path. Thus, a target profile enables you to generate assets for multiple migration path-target platform combinations at the same time for a given Archetype. 
++
```

Comment:
Suggestion for clarity: Change "Through target profiles, you can...' to "You can run more than one generator through a single target profile, with generator..." 
Don't capitalize Archetype here, unless it's capitalized consistently throughout the document.